code/mob/living.dm

```diff
@@ -1638,13 +1638,14 @@ var/global/icon/human_static_base_idiocy_bullshit_crap = icon('icons/mob/human.d
 	var/oldbloss = get_brute_damage()
 	var/oldfloss = get_burn_damage()
 	..()
-	var/newbloss = get_brute_damage()
-	var/damage = ((newbloss - oldbloss) + (get_burn_damage() - oldfloss))
-	if (reagents)
-		reagents.physical_shock((newbloss - oldbloss) * 0.15)
-
-	if ((damage > 0) || W.force)
-		src.was_harmed(M, W)
+	SPAWN_DBG(0.1 SECONDS) //fix race condition
```

Comment:
what race condition is this fixing?

Reply:
`attack` is wrapped in a spawn(0) in the parent